.pre-commit-config.yaml

```diff
@@ -11,7 +11,7 @@
 repos:
   # Autoformat: markdown, yaml
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v4.0.0-alpha.8
+    rev: v3.1.0
```

Comment:
Oops this was not intentionally added, but let's not revert it because it was causing issues for me using the v4 version of prettier - the pre-commit hooks failed if committing no file processed by prettier.